docs/gettingstarted/query_test_report.rst

```diff
@@ -45,6 +45,17 @@ You may find it useful to fetch all failed records and determine pass/fail crite
 
 .. _report_format_fields:
 
+Detailed Reports (``buildtest report --detailed``)
+---------------------------------------------------
+
+The **buildtest report --detailed** option can be used to show a detailed test report that may be of interest when
+examining a test. This option is synonymous to running ``buildtest report --format name, id, user, state, returncode,
```

Comment:
Remove the space between each format field

Reply:
fixed